source/ChromeDevTools/ChromeProcessFactory.cs

```diff
@@ -1,25 +1,31 @@
-﻿using System.Diagnostics;
+﻿using System;
+using System.Diagnostics;
 using System.IO;
 
 namespace MasterDevs.ChromeDevTools
 {
     public class ChromeProcessFactory : IChromeProcessFactory
     {
+        public IDirectoryCleaner DirectoryCleaner { get; set; }
+
+        public ChromeProcessFactory(IDirectoryCleaner directoryCleaner)
+        {
+            DirectoryCleaner = directoryCleaner;
+        }
+
         public IChromeProcess Create(int port)
         {
             string path = Path.GetRandomFileName();
             var directoryInfo = Directory.CreateDirectory(Path.Combine(Path.GetTempPath(), path));
-            var remoteDebuggingArg = "--remote-debugging-port=" + port;
-            var userDirectoryArg = "--user-data-dir=\"" + directoryInfo.FullName + "\"";
-            var chromeProcessArgs = remoteDebuggingArg + " " + userDirectoryArg + " --bwsi --no-first-run";
+            var remoteDebuggingArg = $"--remote-debugging-port={port}";
+            var userDirectoryArg = $"--user-data-dir=\"{directoryInfo.FullName}\"";
+            const string headlessArg = "--headless --disable-gpu";
```

Comment:
Are we forcing headless here?  I'm not comfortable doing this, unless others feel strongly?

Reply:
I'll add an option.